### PR TITLE
changing KB accelerator for settings to be CTRL+I on CommandBar page

### PR DESCRIPTION
**Pull request by salmanmkc**
This addresses issue https://github.com/microsoft/Xaml-Controls-Gallery/issues/301

## Motivation and Context
Addressing issue https://github.com/microsoft/Xaml-Controls-Gallery/issues/301 since Windows I opens settings and Control S normally saves

## How Has This Been Tested?
Compiling

## Screenshots (if appropriate):
Before
image

After
image

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/CommandBarPage.xaml [hunk 1]
```diff
@@ -52,7 +52,7 @@
                 <CommandBar.SecondaryCommands>
                     <AppBarButton x:Name="settingsButton" Icon="Setting" Label="Settings">
                         <AppBarButton.KeyboardAccelerators>
-                            <KeyboardAccelerator Modifiers="Control" Key="S" />
+                            <KeyboardAccelerator Modifiers="Control" Key="I" />
```
Comment (chingucoding) [inline XamlControlsGallery/ControlPages/CommandBarPage.xaml]: This change should also be present in the code sample below (at line 82)

Comment (salmanmkc) [reply XamlControlsGallery/ControlPages/CommandBarPage.xaml]: ah yeah, thanks!

**Comment by stmoy** [comment]
Thank you @salmanmkc for submitting this fix!

**Comment by salmanmkc** [comment]
> Thank you @salmanmkc for submitting this fix!

My pleasure! 😊

**Pull request merged**